pkg/k8s/apis/cilium.io/client/register.go

```diff
@@ -399,22 +240,15 @@ func createCNCCRD(clientset apiextensionsclient.Interface) error {
 // code.
 func createUpdateCRD(
 	clientset apiextensionsclient.Interface,
-	crdName string,
 	crd *apiextensionsv1.CustomResourceDefinition,
 	poller poller,
 ) error {
-	scopedLog := log.WithField("name", crdName)
+	scopedLog := log.WithField("name", crd.Name)
 
 	if !k8sversion.Capabilities().APIExtensionsV1CRD {
```

Comment:
I'm pretty sure that this check can go away; can you confirm and, if reasonable, remove it too?

Reply:
good point.

should be possible to remove since the latest k8s release supported by stable cilium is k8s 1.16 (https://docs.cilium.io/en/stable/concepts/kubernetes/compatibility/#kubernetes-compatibility) and since k8s 1.16 CRD v1 is supported (https://kubernetes.io/docs/reference/using-api/deprecation-guide/#customresourcedefinition-v122).

going to remove this check and its logic